__mocks__/fs-extra.js

```diff
@@ -0,0 +1,66 @@
+const fs = require("fs");
+const path = require("path");
+
+exports.remove = jest.fn();
+exports.ensureDir = jest.fn();
+exports.appendFile = jest.fn();
+exports.copy = jest.fn();
+
+let mockedFiles = {};
+exports.readFileSync = jest.fn();
+
+exports.mockMainPackageJson = (__dirname, object) => {
```

Comment:
This and the following two functions are not really mocking fs-extra, but are providing util methods for the tests so they don't have to think about the file system as much.